DistantWorlds2.ModLoader.ModManager/ModManager.cs

```diff
@@ -613,6 +616,31 @@ private static void HideMessageDialog(object o, DWEventArgs dwEventArgs)
 
   public ConcurrentDictionary<string, object> SharedVariables { get; } = new();
 
+  private byte[] CalculateGameDataChecksum()
+  {
+    var checksumHasher = new Crc64(); //TODO: Switch back to XxHash64 once https://github.com/dotnet/runtime/issues/69184 is resolved and released.
+    DataUtils.ComputeFileHash(checksumHasher, "./DistantWorlds2.ModLoader.dll");
+    DataUtils.ComputeFileHash(checksumHasher, "./DistantWorlds2.exe");
+
+    //TODO?: Incorporate base game data and .Net6 Launcher?
+
+    if (_loadOrder != null)
+    {
+      foreach (var mod in _loadOrder)
+      {
+        checksumHasher.Append(mod.Hash);
+      }
+    }
+
+    return checksumHasher.GetCurrentHash();
+  }
+
+  public string Checksum {
```

Comment:
```suggestion
  // Game data / version / state checksum
  public string Checksum {
```
or...
```suggestion
  public string GameDataChecksum {
```
(would require refactoring)

N.B.